### PR TITLE
Add prettier config

### PR DESCRIPTION
**Pull request by Inrixia**
I made this purely to seperate the formatting changes from the feature changes in the other PR's and to simplify the somewhat complex PR situation my multiple additions have caused due to formatting.

This branch is purely for adding the a `.prettier` config and formatting project files for consistency.

Pull requests https://github.com/alvgaona/cloudflare-discord-bot/pull/14, https://github.com/alvgaona/cloudflare-discord-bot/pull/13 and https://github.com/alvgaona/cloudflare-discord-bot/pull/12 are upstream from the commits on this branch.

Changes to styling requested can either be implemented directly here and then manually refactored into the above PR branches or this can be merged and styling changes can be handled as a seperate PR.

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"useTabs": false,
```
Comment (alvgaona) [inline .prettierrc.json:2]: Thanks!

**Review by alvgaona — APPROVED** [review]
LGTM! 😄

**Comment by Inrixia** [comment]
I beleive this PR is ready to be merged.

**Comment by alvgaona** [comment]
> I beleive this PR is ready to be merged.

Now you can pull for the other PRs, right?

**Comment by Inrixia** [comment]
> Now you can pull for the other PRs, right?

Yes I'll refractor those PR branches to be based off the merged changes to here. Then after those two are merged can discuss doing the same for components.

**Pull request merged**